docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,452 @@
+{{% /admonition %}}
+-->
+
+## Dashboards and visualizations
+
+### Disconnect values in time series, trend, and state timeline visualizations
+
+_Generally available in all editions of Grafana._
+
+<!-- Nathan Marrs -->
+
+You can now choose whether to set a threshold above which values in the data should be disconnected. This can be useful in cases where you have sensors that report a value at a set interval, but you want to disconnect the values when the sensor does not respond. This feature complements the existing [connect null values functionality]({{< relref "../panels-visualizations/visualizations/time-series/#connect-null-values" >}}).
+
+To learn more, refer to our [disconnect values documentation]({{< relref "../panels-visualizations/visualizations/time-series/#disconnect-values" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values-examples.png" max-width="750px" caption="Disconnect values in time series, trend, and state timeline visualizations" >}}
+
+### Flamegraph improvements
+
+_Generally available in all editions of Grafana._
+
+<!-- Andrej Ocenas -->
+
+We have added 4 new features to the flamegraph visualization:
+
+- **Sandwich view**: You can now show a sandwich of any symbol in the flamegraph. Sandwich view will show all the callers on the top and all the callees of the symbol on the bottom. This is useful when you want to see the context of a symbol.
+- **Switching color scheme**: You can now switch color scheme between color gradient by the relative value of a symbol or by package name of a symbol.
+- **Switching symbol name alignment**: With symbols with long name it may be problematic to differentiate them if they have the same prefix. This new option allows you to align the text to left or right so that you can see the part of the symbol name that is important.
+- **Improved navigation**: You can also highlight a symbol or switch on sandwich view for a symbol from the table. Also, a new status bar on top of the flamegraph gives you an overview of which views are enabled.
+
+{{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
+
+### Transformations redesign
+
+The transformations tab has gotten an improved user experience and visual redesign! Now you can view transformations with categories and illustrations.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-transformations.png" max-width="750px" caption="Transformations redesign" >}}
+
+### Format Time Transformation (Alpha)
+
+<!-- Kyle Cunningham -->
+
+_Generally available in all editions of Grafana._
+
+When working with times it can be useful to have different time formats. The new format time transformation allows any time format supported by Moment.js to be used in displaying times. When used in conjunction with the _Group by Value_ transformation this can also be used to bucket days, weeks, and other time windows together.
```

Comment:
```suggestion
When working with times it can be useful to have different time formats. The new format time transformation allows any time format supported by Moment.js to be used in displaying times. When used in conjunction with the _Group by Value_ transformation, this can also be used to bucket days, weeks, and other time windows together.
```